src/database/data/tableModels.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020-2021 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/fluid-project/preferencesServer/blob/main/LICENSE
+ */
+"use strict";
+
+var fluid = require("infusion");
+require("../src/js/index.js");
+
+fluid.registerNamespace("fluid.postgresdb");
+
+fluid.postgresdb.tableDefinitions = `
```

Comment:
Could/should these be stored in plain .sql text files rather than be embedded in code?

Reply:
They could and it should be supported.  Storing in `.js` files makes it easy to load using `require()`, and allows JavaScript template strings.  But, I can see the need for inputting raw SQL from a `.sql` file.

I've looked through `node-postgres` and can't find anything like a `readSqlFromFile()`, so, I'll add that as a method.  It should be short with a call to `fs.readFile()` or `fs.readFileSync()`.  Maybe better is a `runSqlFromFile()` method that loads the SQL and then executes it.